docs/install.md

```diff
@@ -1,26 +1,167 @@
-# Running k0s as a service
+- [Download the k0s binary](#download-the-k0s-binary)
+  - [Prerequisites](#prerequisites)
+    - [Shell script](#shell-script)
+    - [cURL](#curl)
+- [Installing k0s as a service on the local system](#installing-k0s-as-a-service-on-the-local-system)
+  - [Under the hood](#under-the-hood)
+  - [Additional Documentation](#additional-documentation)
 
-Sub-command `k0s install` allows users to easily install k0s as a service, and define its logging.
 
-## Caveats
-* This command is strictly a helper command. It is not meant to provide a fully-automated solution, since you can run k0s in multiple, very different ways.
-* It configures your service set-up as either a worker or a controller, and will have different tasks, depending on the role you pick.
-* Supported services: OpenRC & Systemd
+# Download the k0s binary
 
-## Controller setup
-This is the default mode of operation. When a controller role is picked, the installer will do the following:
+## Prerequisites
 
-* Create user accounts for the different components (see https://github.com/k0sproject/k0s/blob/main/pkg/apis/v1beta1/system.go#L6)
-* Create a service file (OpenRC/Systemd) and redirects logging to `/var/log/k0s.log`.
-* If the `--debug` flag is used, it will also pass this flag along to the service file.
+* https://curl.se/ 
 
-## Worker Setup
-* A worker cannot run with any other user, other than `root`, so no special users will be created.
-* The service file will include the `--token-file` flag, with a value that needs to be manually changed.
-* If the `--debug` flag is used, it will also pass this flag along to the service file.
+Before proceeding any further check out [System Requirements](system-requirements.md)
 
-## Single-node setup
-* Single-node configuration can be installed with 'k0s install controller --enable-worker' command.
+### Shell script
+```sh
+curl -sSLf https://get.k0s.sh | sh
+```
+The shell script accepts following environment variables:
+
+1. `K0S_VERSION=v0.11.0` which allows you to pick k0s version v0.11.0
+2. `DEBUG=true` Prints commands and their arguments as they are executed.
+
+### cURL
+
+This example is essentially what is described in section [above](#shellscript) with the difference that the binary that is downloaded will keep the name as it is and that `chmod` needs to be set to `755` or `+x`
+```
+curl -LJO  "https://github.com/k0sproject/k0s/releases/download/v0.11.0/k0sctl-linux-x64"
+```
```

Comment:
Why do we need this section, if the recommended method is by using get.k0s.sh? I think it's redundant,